app/src/org/commcare/fragments/connect/ConnectJobsMyListFragment.java

```diff
@@ -0,0 +1,233 @@
+package org.commcare.fragments.connect;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.commcare.dalvik.R;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.NavDirections;
+import androidx.navigation.Navigation;
+import androidx.recyclerview.widget.DividerItemDecoration;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+/**
+ * Fragment for showing the "My Jobs" list
+ *
+ * @author dviggiano
+ */
+public class ConnectJobsMyListFragment extends Fragment {
+    public ConnectJobsMyListFragment() {
+        // Required empty public constructor
+    }
+
+    public static ConnectJobsMyListFragment newInstance() {
+        return new ConnectJobsMyListFragment();
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View view = inflater.inflate(R.layout.fragment_connect_my_jobs_lists, container, false);
+
+        RecyclerView recyclerView = view.findViewById(R.id.my_jobs_list);
+
+        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(getContext());
+        recyclerView.setLayoutManager(linearLayoutManager);
+
+        recyclerView.addItemDecoration(new DividerItemDecoration(getContext(), linearLayoutManager.getOrientation()));
+
+        recyclerView.setAdapter(new MyJobsAdapter());
+
+        return view;
+    }
+
+    private static class MyJobsAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
This should probably be in a separate file as `JobsAdapter` which can be used both for all jobs and my jobs tabs.